lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -359,6 +360,9 @@ def write(chunk)
             if record.key?('kubernetes')
               handle_container_metadata(record, entry)
             end
+            # Save the timestamp if available
+            timestamp = nil
+            timestamp = record['time'] if record.key?('time')
```

Comment:
I don't know what I was thinking. I've changed this now to use the ternary operator.